pkg/pluginmanager/manager.go

```diff
@@ -1055,36 +1055,43 @@ func SyncPlugins() error {
 
 // SyncPluginsForContextType installs the plugins for given ContextType
 func SyncPluginsForContextType(contextType configtypes.ContextType) error {
-	ctx, err := configlib.GetActiveContext(contextType)
-	if err != nil {
-		return err
-	}
-	if ctx == nil {
-		return fmt.Errorf(errorNoActiveContexForGivenContextType, contextType)
-	}
-	log.Infof("Checking for required plugins for context %s...", ctx.Name)
+	plugins, err := DiscoverPluginsForContextType(contextType)
 	errList := make([]error, 0)
-	plugins, err := DiscoverServerPluginsForGivenContexts([]*configtypes.Context{ctx})
 	if err != nil {
 		errList = append(errList, err)
 	}
-	err = installDiscoveredContextPlugins(plugins)
+	err = InstallDiscoveredContextPlugins(plugins)
 	if err != nil {
 		errList = append(errList, err)
 	}
 	return kerrors.NewAggregate(errList)
 }
 
-// installDiscoveredContextPlugins installs the given context scope plugins
-func installDiscoveredContextPlugins(plugins []discovery.Discovered) error {
-	var errList []error
-	var err error
-	var installedPlugins []cli.PluginInfo
-	if installedPlugins, err = pluginsupplier.GetInstalledServerPlugins(); err == nil {
+func DiscoverPluginsForContextType(contextType configtypes.ContextType) ([]discovery.Discovered, error) {
+	ctx, err := configlib.GetActiveContext(contextType)
+	if err != nil {
+		return nil, err
+	}
+	if ctx == nil {
+		return nil, fmt.Errorf(errorNoActiveContexForGivenContextType, contextType)
+	}
+	log.Infof("Checking for required plugins for context %s...", ctx.Name)
```

Comment:
Can you put single quotes around the context name so that it matches the other printouts?